File: Arrays/1662. Check If Two String Arrays are Equivalent.js
link = 'https://leetcode.com/problems/check-if-two-string-arrays-are-equivalent/'


// Solution
//---------------------------------
const arrayStringsAreEqual = (word1, word2) => {
    let str1 = word1.join('')
    let str2 = word2.join('')
    if(str1===str2){
        return true
    }else{
        return false
    }
}
//---------------------------------
let result = arrayStringsAreEqual(["ab", "c"],["a", "bc"])
console.log(result)

File: Arrays/1662. Check If Two String Arrays are Equivalent.py
def arrayStringsAreEqual(word1, word2):
    str1 = "".join(word1)
    str2 = "".join(word2)
    if str1==str2:
        return True
    else:
        return False

print(arrayStringsAreEqual(["ab", "c"],["a", "bc"]))